test/statistics/test_ranks.py

```diff
@@ -0,0 +1,113 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from collections import OrderedDict
+
+import numpy as np
+import pytest
+import torch
+
+from earth2studio.statistics import rank_histogram
+
+
+@pytest.mark.parametrize(
+    "ensemble_dimension",
+    [
+        "ensemble",
+        "time",
+    ],
+)
+@pytest.mark.parametrize("device", ["cpu", "cuda:0"])
+def test_rank_histogram(ensemble_dimension: str, device: str) -> None:
```

Comment:
how hard is it to make a test that checks the ranked histogram converges to a uniform distribution, say you make a bin count of 2 or 3. I imaging a few hundred samples would get you close?

Reply:
Done.